packages/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/ReanimatedPackage.java

```diff
@@ -36,7 +36,7 @@
       ReanimatedModule.class,
       ReanimatedUIManager.class,
     })
-public class ReanimatedPackage extends TurboReactPackage implements ReactPackage {
+public class ReanimatedPackage extends BaseReactPackage implements ReactPackage {
```

Comment:
`BaseReactPackage` already implements `ReactPackage` so the implements is unnecessary.